### ISSUE TITLE
Fix Request: false positive on Kotlin AvoidMultipleConcatStatements for mutableMap

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#674]
```
private fun falsePos(): Map<String, String> {
        val key = "key:"
        return mutableMapOf<String, String>().apply {
            this[key + "A"] = "valA"
            this[key + "B"] = "valB" // <-- false positive
        }
}
```

**Comment by stokpop** [comment]
see also #561
